### ISSUE TITLE
dlt-viewer installation step fails

### ISSUE THREAD

**Issue by gunnarx** [Pelagicore/pelux-sde#15]
dlt-viewer installation step fails

**To reproduce:**
```vagrant up```
on a new fresh install.

**Result:**

```
    default: Running provisioner: shell...
    default: Running: /tmp/vagrant-shell20180425-11575-1g8sfb6.sh
    default: Cloning into 'dlt-viewer'...
    default: fatal: reference is not a tree: 3f25fe124636239d0cc8fbf043a404f4b6a81668
```

I could find this hash locally in my repository. It is not in the upstream GENIVI repo anymore apparently, so some dlt-viewer maintainers must have force-pushed master?  From what I can tell anyway...  I might ask them later but it is what it is now.  Just updating dlt-viewer to something newer should probably fix this.

**Comment by GordanM** [comment]
@TSAnsari could you please verify this?

**Comment by TSAnsari** [comment]
Yes I can see the CI job fails for sde due to dlt-viewer not being installed.
http://pelux.io/jenkins/blue/organizations/jenkins/ci-pelux-sde/detail/PR-12/150/pipeline

**Comment by GordanM** [comment]
@TSAnsari I believe if you just take the top revision from the current `master`it should be fine.

**Comment by TSAnsari** [comment]
yup will do that

**Comment by TSAnsari** [comment]
I have now changed the revision to the latest on master. But now it seems that when building dlt-viewer, it does not find the serialport module installed. From https://wiki.qt.io/Qt_Serial_Port it is mentioned that from qt5.1 and onwards, serialport is part of qt and need not be build/installed separately. However, for some reason it is not being installed on the VirtualBox, hence failing test. Investigating further. @rpannek Did you see the same issue when running these sde tests in your project?

**Comment by frznlogic** [comment]
@TSAnsari have you tried installing libqt5serialport5 also in the Vagrantfile?

**Comment by TSAnsari** [comment]
@frznlogic Yes I did and it works now. For some reason it didn't installed serialport although the above document from Qt mentions that it should be installed along with Qt.

**Comment by GordanM** [comment]
Will be fixed with #12 .
